acquia/acquia_cms
# ACMS-853: Refactor root composer.json file and move it into Acquia CMS individual modules

**Motivation**

Fixes #[ACMS-853](https://backlog.acquia.com/browse/ACMS-853)

**Proposed changes**

Remove root composer dependency and move it to module's composer file and make sure it exists in module's composer.json.

**Alternatives considered**

NA

**Testing steps**


- Test site install with acquia_cms and run the local test suites and make sure everything is working as expected
- Test site install with vanilla drupal and run the local test suites and make sure everything is working as expected

**Merge requirements**
